### DISCUSSION TITLE
xdg-desktop-portal-impl which to choose?

### DISCUSSION THREAD

**Discussion by miranyx** [hyprwm/Hyprland#1883] [Q&A] [answered]
Greetings,

Upon attempting to install xdg-desktop-portal-hyprland-git on Arch Linux using yay -Syu xdg-desktop-portal-hyprland-git, I have been prompted to install xdg-desktop-portal-impl. I am uncertain as to which option I should select. Could you please advise me on this matter?

![20230326_10h30m20s_grim](https://user-images.githubusercontent.com/64905406/227764424-efcde43b-49b1-4277-9cc1-fc1f6b086d82.png)

**Comment by vaxerski** [comment, marked as answer]
@ThatOneCalculator can you fix this for god's sake

select gtk, the only that doesnt cause issues

**Reply by ThatOneCalculator** [reply]
It provides xdg-desktop-portal-imp, not sure what else I could do...

![image](https://user-images.githubusercontent.com/44733677/227796183-1e63b6ab-c9cc-4980-9435-7078d4283a36.png)

**Reply by vaxerski** [reply]
add to `provides` and `conflicts` `xdg-desktop-portal-wlr` and it should be ok

we conflict with wlr anyways

**Reply by ThatOneCalculator** [reply]
pushed that
